kolibri_explore_plugin/assets/src/views/ChannelsPage.vue

```diff
@@ -112,6 +134,12 @@
           name: PageNames.SEARCH,
         });
       },
+      goToTerm(event) {
+        this.$router.push({
+          name: PageNames.SEARCH,
+          params: { query: event.target.innerHTML.trim() },
```

Comment:
Instead of using `innerHTML` you can pass the term as a parameter to the `goToTerm()` handler.